server/app/views/applicant/EditTiClientView.java

```diff
@@ -57,30 +57,39 @@ public Content render(
       ApplicantPersonalInfo personalInfo,
       Http.Request request,
       Messages messages,
-      Long accountId,
-      Optional<Form<EditTiClientInfoForm>> editTiClientInfoForm,
-      boolean isSuccessfulSave) {
-
+      Optional<Long> accountIdToEdit,
+      Long applicantIdOfTi,
+      Optional<Form<TiClientInfoForm>> tiClientInfoForm) {
+    Optional<AccountModel> optionalAccountModel = Optional.empty();
+    String title = "Add new Client";
+    String pageHeader = "Add client";
```

Comment:
suggestion (blocking):  Could you please use the localized message string here and also for the "Edit client" page header text on line 72?

Reply:
sure!

Reply:
There is no title "Edit client information". There is no page header for either. I see this messages.at(MessageKey.TITLE_CREATE_CLIENT.getKeyName()); so updated this one.